views/new.erb

```diff
@@ -0,0 +1,8 @@
+<h1>Hello</h1>
+<form action="/chitter" method="post">
+  <h2>Enter your post</h2>
+  <input type="text" name="message" />
+  <h2>Enter the time</h2>
+  <input type="text" time="time" />
```

Comment:
I can see where you are going with this - would the user need to supply the time? Or is this something that could happen at the database or server level?